library/core/src/main/java/com/google/android/exoplayer2/text/ttml/TtmlNode.java

```diff
@@ -195,6 +236,24 @@ private void getEventTimes(TreeSet<Long> out, boolean descendsPNode) {
     return cues;
   }
 
+  private void traverseForImage(
+      long timeUs,
+      String inheritedRegion,
+      List<Pair<String, String>> regionImageList) {
+
+    String resolvedRegionId = ANONYMOUS_REGION_ID.equals(regionId) ? inheritedRegion : regionId;
+
+    if (isActive(timeUs)) {
+      if (TAG_DIV.equals(tag) && imageId != null) {
+        regionImageList.add(new Pair<>(resolvedRegionId, imageId));
```

Comment:
Question: Is it OK to return here? Or do we really need to traverse (L252-254) even though we've found an image at this node?